packages/drivers/routerlicious-driver/src/documentStorageService.ts

```diff
@@ -225,9 +230,22 @@ export class DocumentStorageService implements IDocumentStorageService {
         const hash = await gitHashFile(IsoBuffer.from(parsedContent, encoding));
         if (!this.blobsShaCache.has(hash)) {
             this.blobsShaCache.set(hash, "");
-            const blob = await this.manager.createBlob(parsedContent, encoding);
+            const blob = await this.manager.createBlob(parsedContent, encoding)
+                .catch(DocumentStorageService.enhanceGitServiceError);
             assert(hash === blob.sha, "Blob.sha and hash do not match!!");
         }
         return hash;
     }
+
+    public static async enhanceGitServiceError(error: any): Promise<never> {
+        const messageFallback = "GitManager call failed";
+        // GitManager's Historian's RestWrapper only throws status code when response is available
```

Comment:
Do we need to make historians error response better?

Reply:
Yes, I think improving RestWrapper's error response would be helpful. We lose a lot of info when the errors are parsed down to just a status code